src/analyzer/protocol/ssl/tls-handshake.pac

```diff
@@ -8,6 +8,7 @@
 
 #include "zeek/analyzer/protocol/ssl/types.bif.h"
 #include "zeek/analyzer/protocol/ssl/events.bif.h"
+#define USE_FLIPPED
```

Comment:
Why do we need this define?

Reply:
So....
`proc-certificate.pac` is included by both the ssl analyzer (for SSLv2) and the SSL handshake analyzer (for SSLv3 up). The tracking takes place in the SSL handshake analyzer. So - in case this is called in the context of the handshake analyzer, you can just access the local variable. In case of the SSL analyzer, you have to call the function that calls into the SSL handshake analyzer to get the variable.

I was not able to come up with a better way that does not have tons of code duplication. I at first tried to do it with c++ metaprogramming - but I couldn't get that to work - and it started looking like a horrible mess.

I am open to better ideas though :)

Reply:
Oh, yes - and the define here is to indicate that this is the TLS handshake analyzer that has the `flipped_` variable available. Proc-certificate uses that to just use it, if the define is present (which is it, because it is included indirectly from this file).

On the plus side - this is not easy to mess up. If you remove this, the build breaks.

Reply:
Ah, ok, now I also get the comment. :) 

This is in ` proc_certificate`, could you pass `flipped_` in as a parameter? If not, then I don't have a better idea either, fine with me.

Reply:
Yup, that works. I... for some reason did not think about that. I am going to update this PR, let the testsuite run, and then merge.